src/plugins/data_views/public/saved_objects_client_wrapper.ts

```diff
@@ -21,35 +17,49 @@ import {
   SavedObjectsClientCommonFindArgs,
 } from '../common/types';
 
-type SOClient = Pick<
-  SavedObjectsClientContract,
-  'find' | 'resolve' | 'update' | 'create' | 'delete'
->;
-
-const simpleSavedObjectToSavedObject = <T>(simpleSavedObject: SimpleSavedObject): SavedObject<T> =>
-  ({
-    version: simpleSavedObject._version,
-    ...omit(simpleSavedObject, '_version'),
-  } as SavedObject<T>);
+import type {
+  DataViewGetIn,
+  DataViewGetOut,
+  DataViewCreateIn,
+  DataViewCreateOut,
+  DataViewUpdateIn,
+  DataViewUpdateOut,
+  DataViewDeleteIn,
+  DataViewDeleteOut,
+  DataViewSearchIn,
+  DataViewSearchOut,
+  // DataViewSearchQuery,
+} from '../common/content_management';
 
 export class SavedObjectsClientPublicToCommon implements SavedObjectsClientCommon {
-  private savedObjectClient: SOClient;
+  private contentManagemntClient: ContentClient;
 
-  constructor(savedObjectClient: SOClient) {
-    this.savedObjectClient = savedObjectClient;
+  constructor(contentManagemntClient: ContentClient) {
+    this.contentManagemntClient = contentManagemntClient;
   }
 
-  async find<T = unknown>(options: SavedObjectsClientCommonFindArgs) {
-    const response = (await this.savedObjectClient.find<T>(options)).savedObjects;
-    return response.map<SavedObject<T>>(simpleSavedObjectToSavedObject);
+  async find(options: SavedObjectsClientCommonFindArgs) {
+    console.log('******* find', options);
+    const results = await this.contentManagemntClient.search<DataViewSearchIn, DataViewSearchOut>({
```

Comment:
@sebelga I must be doing something wrong here.

Reply:
Looks good to me.

Reply:
You need to pass an object for the `query`.

<img width="508" alt="Screenshot 2023-03-27 at 11 20 39" src="https://user-images.githubusercontent.com/2854616/227899406-60577273-9517-4044-a1aa-f7e1978bbf63.png">

Add an empty object if you don't need any parameters

```ts
const results = await this.contentManagemntClient.search<DataViewSearchIn, DataViewSearchOut>({
  contentTypeId: 'index-pattern',
  query: {},
});
```